lgc/test/InOutPackingNonZeroBase.lgc

```diff
@@ -1,7 +1,8 @@
 ; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --tool lgc
 ; RUN: lgc -mcpu=gfx900 --print-after=lgc-patch-resource-collect --verify-ir %s -o=/dev/null 2>&1 | FileCheck --check-prefixes=IR %s
 ; Throw in 'cat' as a hack to prevent update_test_checks from touching the "MAPPING" lines
-; RUN: lgc -mcpu=gfx900 -v --verify-ir %s -o=/dev/null 2>&1 | cat | FileCheck --check-prefixes=MAPPING %s
+; RUN: lgc -mcpu=gfx900 -v --verify-ir %s -o=/dev/null 2>&1 > %t.out
+; RUN: cat %t.out | FileCheck --check-prefixes=MAPPING %s
```

Comment:
I did this because I wanted to use update_test_checks, but I didn't want update_test_checks to touch the check prefixes of that particular RUN line (which does some more unusual matching). The `cat` was enough to throw the tool  off, I suppose this alternative works as well.

Reply:
Ah, I should have read the comment in the line above…

Reply:
Yes, the LIT_USE_INTERNAL_SHELL support wasn't good enough to allow the original (and neater) form - hence the change.